src/jdk.jpackage/windows/classes/jdk/jpackage/internal/resources/WinResources_de.properties

```diff
@@ -57,6 +57,8 @@ error.unlock-resource=Aufheben der Sperre nicht erfolgreich: {0}
 error.read-wix-l10n-file=Datei {0} konnte nicht geparst werden
 error.extract-culture-from-wix-l10n-file=Kulturwert konnte nicht aus Datei {0} gelesen werden
 error.short-path-conv-fail=Failed to get short version of "{0}" path
+error.missing-service-installer='service-installer.exe' service installer not found in the resource directory
+error.missing-service-installer.advice=Add 'service-installer.exe' service installer to the resource directory
```

Comment:
No need to substitute English resouces into localized ones. L10n process will automatically detect the diff, and translate them accordingly.